### PR TITLE
Improvement ImageViewReader don't necessary loadTexture when imageFilePa...

### PR DESCRIPTION
**Pull request by zhouxiaoxiaoxujian**
...th is empty

### CODE REVIEW

File: cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp [hunk 1]
```diff
@@ -41,8 +41,12 @@ namespace cocostudio
         
         const rapidjson::Value& imageFileNameDic = DICTOOL->getSubDictionary_json(options, "fileNameData");
         int imageFileNameType = DICTOOL->getIntValue_json(imageFileNameDic, "resourceType");
-        std::string imageFileName = this->getResourcePath(imageFileNameDic, "path", (Widget::TextureResType)imageFileNameType);
-        imageView->loadTexture(imageFileName, (Widget::TextureResType)imageFileNameType);
+        std::string imageFilePath = DICTOOL->getStringValue_json(imageFileNameDic, "path");
```
Comment (zilongshanren) [inline cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp]: I think we have checked the empty of  `imageFilePath` in the imageView's `loadTexture` method.  Please checkout it .

Comment (zhouxiaoxiaoxujian) [inline cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp]: Thank you for you reply! I’ll checkout it.

Comment (zhouxiaoxiaoxujian) [inline cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp]: Oh! What I mean is , If live 'path' property empty, imageViewRead will load dictionary path, that's may cause error.

Comment (minggo) [inline cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp]: @andyque 
Any update?

Comment (pipu) [inline cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp]: The following section is right, can merge it.

Comment (minggo) [inline cocos/editor-support/cocostudio/WidgetReader/ImageViewReader/ImageViewReader.cpp]: @pipu 
Did you mean this pull request can be merged?

**Comment by zilongshanren** [comment]
@minggo 
Yes, I think it's safe to merge this PR. But it contains merge conflicts.
hi,  @zhouxiaoxiaoxujian  
could you please fix the merge conflicts?

**Comment by zhouxiaoxiaoxujian** [comment]
@andyque  
ok, I rebase from v3 branch, and fix conflicts.

**Comment by zilongshanren** [comment]
@minggo 
It's ok to merge it now.

**Comment by minggo** [comment]
Thanks guys.

**Pull request merged**